chapter_11_ansible.asciidoc

```diff
@@ -318,6 +324,10 @@ for.
 ----
 ====
 
+// JAN: I'm stuck here. Docker is not installed on EC2 instance with Ubuntu. Googling it, it seems like this is some outdated Docker installation process
+The one described here is much closer to Docker's installation guide: https://www.digitalocean.com/community/tutorials/how-to-use-ansible-to-install-and-set-up-docker-on-ubuntu-22-04
```

Comment:
hi Jan, sorry about this, my fault, I've since updated the book to say you need to install `docker.io` not `docker`

(you can also do a bunch of faffing with docker's private package store but i want to avoid that for the book if i can)